### ISSUE TITLE
Suggest translations for conditionals too where relevant

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/eslint-plugin#113]
One more edge case to handle in a follow-up PR is conditionals:

```ts
<button title="Show layer" />                              // reported
<button title={visible ? 'Hide layer' : 'Show layer'} />  // not reported
```

_Originally posted by @krassowski in https://github.com/jupyterlab/eslint-plugin/pull/102#pullrequestreview-5192489742_
